cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,115 @@
 /// <reference types='cypress' />
 
+// const { AggregateError } = require("sequelize");
+
 describe('Web Tables page', () => {
-  it('', () => {
+  const randomIndex = Math.floor(Math.random() * 2);
+  const user = {
+    firstName: 'John',
+    lastName: 'Doe',
+    email: `johndoe${randomIndex}@gmail.com`,
+    age: 24,
+    salary: 2000,
+    department: 'Police'
+  };
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('Validation of pagination', () => {
+    cy.get('.-previous > .-btn').should('be.visible');
+    cy.get('.-next > .-btn').should('be.visible');
+    cy.get('[class="-pageInfo"]').should('be.visible');
+    cy.get('.-pageJump > input').should('contain.value', 1);
+  });
+
+  it('should allow to select row count', () => {
+    cy.get('select').select('5 rows');
+    cy.get('select').should('have.value', 5);
+
+    cy.get('select').select('10 rows');
+    cy.get('select').should('have.value', 10);
+
+    cy.get('select').select('20 rows');
+    cy.get('select').should('have.value', 20);
+
+    // cy.get('select').select('25 rows');
```

Comment:
I think you should uncomment these rows)

Reply:
That's really so, thanks, fixed)